lib/providers/notifiers/connection_notifier.dart

```diff
@@ -13,31 +19,48 @@ class ConnectionNotifier extends ChangeNotifier {
     'https://mainnet.telosusa.io',
     'https://telos.eosphere.io',
     'https://telos.caleos.io',
+    'https://api.eos.miami',
```

Comment:
Miami?

Reply:
Miami is a big block producer in the USA